app/models/patron.rb

```diff
@@ -261,6 +259,12 @@ def can_schedule_special_collections_visit?
 
   private
 
+  def academic_staff_or_fellow?
+    return unless profile_key == 'CNAC'
+
+    affiliations.include?('staff:academic') || affiliations.include?('affiliate:fellow')
```

Comment:
@saseestone just to validate we're are only taking users w/ profile CNAC and affiliation is `staff:academic` (in addition to `affiliate:fellow`) so we exclude users w/ affiliations like `staff:otherteaching` and `staff:emeritus` (as well as a few other cases)?

Reply:
Please add staff:otherteaching.  Thanks!